bin/stacks/routing-database-stack.ts

```diff
@@ -148,7 +153,7 @@ export class RoutingDatabaseStack extends cdk.NestedStack {
       }
     )
 
-    // Creates a Table for storing health state of RPC providers
+    // TODO(jie): Remove this table after we fully migrate to rpcProviderHealthStateDynamoDb
```

Comment:
I would caution removing the table, because we've never explicitly set delete for retention policy on dynamo table:

Codewise, I mean we will have to specify destroy on removal, otherwise it's retain by default:
```diff
this.rpcProviderStateDynamoDb = new aws_dynamodb.Table(this, DynamoDBTableProps.RpcProviderStateDbTable.Name, {
      tableName: DynamoDBTableProps.RpcProviderStateDbTable.Name,
      partitionKey: {
        name: DynamoDBTableProps.RpcProviderStateDbTable.PartitionKeyName,
        type: AttributeType.STRING,
      },
      billingMode: BillingMode.PAY_PER_REQUEST,
      timeToLiveAttribute: DynamoDBTableProps.TTLAttributeName,
+     removalPolicy: RemovalPolicy.DESTROY,
    })
```

Since CDK relies on knowing the state of the DB, removing this DB might cause the next CDK deploy to fail in prod, and we will be stuck.

I recommend we just leave it as is, since we get the DDB bill pay per request.

Reply:
OK I see what you mean. To be safe, I can leave the old table in the stack as it is and add comment saying it's obsolete.